app/assets/stylesheets/gradesheet.css.scss

```diff
@@ -242,8 +242,8 @@ td.id {
 
 
 #grades td .score_details .feedback {
-    width: 290px;
-    min-height: 75px;
+    width: 350px;
```

Comment:
It's large enough to keep standard line-dividers in one line as seen in these examples from 15-112 and 15-122

<img width="394" alt="screen shot 2016-10-04 at 1 37 02 pm" src="https://cloud.githubusercontent.com/assets/1356687/19085618/ef280bfa-8a38-11e6-97f6-a82e192fab2f.png">

<img width="375" alt="screen shot 2016-10-04 at 1 40 35 pm" src="https://cloud.githubusercontent.com/assets/1356687/19085625/f2bd7f2a-8a38-11e6-854c-6bd8bf71f4a4.png">